client/app/bundles/shared/components/Avatar.jsx

```diff
@@ -0,0 +1,28 @@
+// @flow
+import React from 'react';
+import css from './Avatar.scss';
+
+type Props = {
+  src: string,
+  alt: string,
+  name?: string
+};
+
+const Avatar = (props: Props) => {
+  const { name, src, alt } = props;
+  const avatarClassNames = `${css.avatar} ${name ? css.white : ''}`;
+  const nameTag = name ? <p>{name}</p> : '';
```

Comment:
Any reason why this is being wrapped in a `<p>`?

Reply:
I added it in order to wrap the text in a block element. I would never print plain text without a tag! You never know if it will need some styling... haha

Reply:
Is it added extra unnecessary padding/margins? If so I recommend using `<div>`